src/Components/Links/links.js

```diff
@@ -0,0 +1,7 @@
+const Link = ({ children }) => {
+    return(
+        <a className='textDecor white spacement border'>{children}</a>
+    );
+}
```

Comment:
You can return it directly when using arrow functions.

```suggestion
const Link = ({ children }) => (
  <a className='textDecor white spacement border'>{children}</a>
)
```

Reply:
Also you can just move this component to `src/Components/Link/index.js`
Instead of importing from `./Links/links`